### DISCUSSION TITLE
ability to choose the amount of hits shown with ColourHitMeter

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#20036] [Ideas] [answered]
ive been playing around making an only  lazer elements skin, and ive realized this element is rather limited in its current state, would it be of interest to make the spacing in between hits, the shapes, and the number of shown hits settings for the element?

and by of interest i mean im willing to do it myself, just want the go ahead 
example video :

https://user-images.githubusercontent.com/74463310/187548807-4f680f41-66dd-4366-b7c1-2627a33e7a5e.mov

**Comment by peppy** [comment, marked as answer]
Sounds fine.

**Comment by Moltanicaa** [comment]
Attach skin please

**Comment by mk56-spn** [comment]
https://user-images.githubusercontent.com/74463310/188245507-019c285c-7e19-489d-bd1f-8216f0e2f5f9.mov

mostly done, ran into an issue when adding shapes. for one theres very few normal ones, and using font awesome results in horrid blurry icons if you even start scaling them a little

**Reply by Feodor0090** [reply]
Can you add rounding radius customization?
For ability to have squares / rounded squares, not just ideal circles.

**Reply by mk56-spn** [reply]
yeah im trying something, ill see if i can work it out
